### ISSUE TITLE
Travis build fails

### ISSUE THREAD

**Issue by ejoerns** [rauc/rauc-hawkbit#6]
```
$ tox-travis
tox-travis: command not found
```

**Comment by ejoerns** [comment]
targeted outdated branch. Thus no action needed

**Comment by ejoerns** [comment]
And we can instantly recycle this one as Travis really fails (but with a different issue):
```
>       with aiohttp.Timeout(self.timeout):
E       AttributeError: module 'aiohttp' has no attribute 'Timeout'
```

**Comment by Bastian-Krause** [comment]
`aiohttp.Timeout` was removed in aiohttp-3.0.0 (see [changelog referencing PR #2348](https://docs.aiohttp.org/en/stable/changes.html#id116))

We should use the timeout keyword of `aiohttp.ClientSession.get()` instead as [described here](https://docs.aiohttp.org/en/stable/client_quickstart.html#timeouts).

**Comment by Bastian-Krause** [comment]
I will send a PR fixing this.

**Comment by Bastian-Krause** [comment]
PR #7 and #9 are intended to fix this.

**Comment by ejoerns** [comment]
Fixed by #7 and #8
